File: __tests__/__fixtures__/output/javascript/jquery/multipart-form-data-no-params.js
const settings = {
  "async": true,
  "crossDomain": true,
  "url": "https://httpbin.org/anything",
  "method": "POST",
  "headers": {
    "Content-Type": "multipart/form-data"
  }
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: src/targets/swift/urlsession/fixtures/multipart-form-data-no-params.swift
import Foundation

let url = URL(string: "https://httpbin.org/anything")!
var request = URLRequest(url: url)
request.httpMethod = "POST"
request.timeoutInterval = 10
request.allHTTPHeaderFields = ["Content-Type": "multipart/form-data"]

let (data, response) = try await URLSession.shared.data(for: request)
print(String(decoding: data, as: UTF8.self))